changelogs/fragments/777-add_ability_to_identify_ed25519_complete_chains.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - certificate_complete_chain - add ability to identify ed25519 and ed448 complete chains (https://github.com/ansible-collections/community.crypto/pull/777).
```

The algorithms are called `EdXXX`, with an upper-case `E`:
```suggestion
  - certificate_complete_chain - add ability to identify Ed25519 and Ed448 complete chains (https://github.com/ansible-collections/community.crypto/pull/777).
```